### PR TITLE
Style the home page and improve team CSS

### PR DESCRIPTION
**Pull request by KrishKittur**

### CODE REVIEW

**Comment by AnnikaCodes** [comment]
Are the extra changes to team.tsx intended?

**Comment by KrishKittur** [comment]
> Are the extra changes to team.tsx intended?

Yes

File: src/index.ts [hunk 1]
```diff
@@ -116,13 +116,46 @@ if (Config.nosecurity) {
 }
 
 server.get('/', async (req: AuthenticatedRequest, res) => {
-    let html = `Hello! This is the FRC Team ${Config.teamNumber || 5940} scouting website.`;
+    let html = ``
+    ;
```
Comment (TheAnnalyst) [inline src/index.ts:120]: What's this about?

File: src/index.ts [hunk 2]
```diff
@@ -116,13 +116,46 @@ if (Config.nosecurity) {
...
+      html += `<html>
+        <head>
+          <title>Home</title>
+          <link rel="stylesheet" href="css/home.css">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        </head>
+        <body><div class="logged-in-content">
+          <div class="text-group">
+            Hello! This is the FRC Team 5940 scouting website.
+            <p>You are logged in as Krish Kittur`;
```
Comment (TheAnnalyst) [inline src/index.ts:132]: What is this? You shouldn't be hardcoding your name; you can see how I used `${(req.oidc.user as any).name || 'an unknown user'}` above. Also, you forgot to close the `<p>` tag.

File: src/index.ts [hunk 3]
```diff
@@ -116,13 +116,46 @@ if (Config.nosecurity) {
...
+      html += `
```
Comment (TheAnnalyst) [inline src/index.ts:133]: You're using multiline template strings; why two `+=` statements? Either each line can be its own `+=`, or make one statement.

File: src/index.ts [hunk 4]
```diff
@@ -116,13 +116,46 @@ if (Config.nosecurity) {
...
+            <li><a href="/addmatch"><div class="button">Add Match</div></a></li>
+            <li><a href="/viewteam"><div class="button">View Team</div></a></li>
+            <li><a href="/viewmatch"><div class="button">View Match</div></a></li>
+          </ul>
+        </div></body>
+      </html>`;
     } else {
-        html += `<p>You will need to <a href="/login">log in</a> to use this service.</p>`;
+        html += `
+        <html>
```
Comment (TheAnnalyst) [inline src/index.ts:146]:
```suggestion
        html += `<html>
```

File: src/templates/team.tsx [hunk 5]
```diff
@@ -15,47 +15,98 @@ const properties = Object.keys(dummyMatch)
...
             try {
                 const mean = team.getMean(stat as keyof InfiniteRecharge.InfiniteRechargeMatch);
-                statHTML = <span>
-                    Mean {normalizePropertyName(stat)}: {mean}
-                </span>;
+                statHTML = <div class="stat-container" id="stat-container-1">
+                  <p class="stat-label">Mean{`${normalizePropertyName(stat)}: ${mean}`}</p>
+                </div>;
+
+
```
Comment (TheAnnalyst) [inline src/templates/team.tsx:73]:
```suggestion
```

File: src/templates/team.tsx [hunk 6]
```diff
@@ -15,47 +15,98 @@ const properties = Object.keys(dummyMatch)
...
+                      team.matches
+                          .map((match) => <li><a href="/viewmatch?match={match.number}"><button>Match {match.number}</button></a></li>)
+                          .join('')
+                  }</ul>
+                </div>
+              </div>
+              <script> </script>
+          </body>
+        </html>;
+
```
Comment (TheAnnalyst) [inline src/templates/team.tsx:110]:
```suggestion
```

**Review by TheAnnalyst — CHANGES_REQUESTED** [review]
Do you mind sending me screenshots of what this looks like?

File: src/index.ts (hunk 2 above)
Comment (KrishKittur) [reply src/index.ts:132]: Oh my b, I did that because it wouldn't compile otherwise and I forgot to change it back.

File: src/index.ts (hunk 3 above)
Comment (KrishKittur) [reply src/index.ts:133]: Yeah it would probably be better to stick them together.

Comment (KrishKittur) [reply src/index.ts:133]: That's how it was before, and I never bothered to change it lol.

File: src/index.ts (hunk 1 above)
Comment (TheAnnalyst) [reply src/index.ts:120]: (It should probably be all one line)

**Review by TheAnnalyst — APPROVED** [review]
Actually, never mind; I'll just merge this and fix any issues later.

File: src/index.ts [hunk 7]
```diff
@@ -116,13 +116,44 @@ if (Config.nosecurity) {
...
+      html += `<html>
+        <head>
+          <title>Home</title>
+          <link rel="stylesheet" href="css/home.css">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        </head>
+        <body><div class="logged-in-content">
+          <div class="text-group">
+            Hello! This is the FRC Team ${Config.teamNumber || 5940} scouting website.
+            <p>You are logged in as You are logged in as ${(req.oidc.user as any).name
```
Comment (TheAnnalyst) [inline src/index.ts:131]:
```suggestion
            <p>You are logged in as ${(req.oidc.user as any).name || 'an unknown user'}. 
```
lmao, idk what happened here

**Pull request merged**